bindings/go/Makefile.am

```diff
@@ -5,11 +5,12 @@
 
 if HAVE_GO
 
-GOPATH=$(abs_top_builddir)/bindings/go/$(objdir)
+GOOBJDIR=$(abs_top_builddir)/bindings/go/$(objdir)
+GOPATH=$(abs_top_srcdir)/bindings/go/
```

Comment:
maybe 
`GOOBJDIR=$(GOPATH)/$(objdir)`

Reply:
`GOPATH=$(abs_top_srcdir)` we don't want the output to be in source dir, but in build dir when we using out of source build.

Reply:
right, missed it